swo/mpt/extensions/runtime/djapp/conf/default.py

```diff
@@ -165,6 +166,13 @@
     },
 }
 
+
+# OpenTelemetry configuration
+SERVICE_NAME = os.getenv("SERVICE_NAME", "Swo.Extensions.AdobeVIPM")
```

Comment:
Probably we should think about something like:
Swo.Extensions.<unique_extension_id>
In any case, please change this default to some generic value not related to adobe.

Reply:
Well for today we have names of the service for that, but definitely - yes